concent_api/common/exceptions.py

```diff
@@ -20,3 +20,7 @@ def __init__(self, error_message: Optional[str], error_code: ErrorCode) -> None:
         self.error_code = error_code
         self.error_message = '' if error_message is None else error_message
         super().__init__(error_message)
+
+
+class ConcentValidationError(ConcentBaseException):
```

Comment:
Please unify the name and location with @dybi as he is doing something similar in https://github.com/golemfactory/concent/pull/608.

Reply:
Ok, we agree on leaving this one.

Reply:
Please move to `core.exceptions` if you are using it only in `core` module

Reply:
Already did it ;)